src/devices/MAX31856/samples/README.md

```diff
@@ -0,0 +1,39 @@
+## Usage
+The MAX31856.samples file contains a sample usage of the device. Note that this reads two temperatures. One is a connected thermocouple reading which can be read using the  ```TryGetTemperature``` command and the other is the temperature of the device itself which can be read using the ```GetColdJunctionTemperature``` command. The Cold Junction Temperature is used internally to increase the accuracy of the thermocouple but can also be read if you find a use for it.
```

Comment:
- extra enter after header
- header should be `#` rather than `##`

Reply:
updated